frontend/src/components/DynamicDiv/index.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import Button from "../Button";
+
+export default function Div({ style}) {
+  const [state, setState] = useState("Initial Text");
+  return (
+    <>
+      <div className="DynamicDiv" style={style}>
+        {state}
+      </div>
+      <Button
+        onClick={() => { 
+          setState("New text");
+        }}>
+        Change Text
```

Comment:
You already know this, but you can use the {child} prop like in Dat's button here so that you can decide what you want the text to say for each time you call your Div function.